ambry-replication/src/main/java/com.github.ambry.replication/DiskTokenPersistor.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.replication;
+
+import com.github.ambry.store.FindToken;
+import com.github.ambry.utils.CrcOutputStream;
+import com.github.ambry.utils.SystemTime;
+import java.io.DataOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * {@link DiskTokenPersistor} persists replication token to disk.
+ */
+class DiskTokenPersistor implements ReplicaTokenPersistor {
+
+  private Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
private static final

Reply:
getClass() is not support in a static context. `final` will be used.

Reply:
The logger is generally declared static, probably so static methods can also use it.  It's okay to hardcode the class name instead of getClass().